src/components/NotificationList/NotificationList.module.css

```diff
@@ -0,0 +1,92 @@
+@import '../../design-tokens/mixins.css';
+/*------------------------------------*\
+    #NOTIFICATION LIST
+\*------------------------------------*/
+
+/**
+ * 1)  Notification list contains notification items, a count of unread notifications, and a button to mark all notifications as read
+ */
+.notification-list {
```

Comment:
Let's add `@mixin reset` to get rid of any padding and margin caused by the `ul` tag

Reply:
thanks!!